plenum/test/view_change/helper.py

```diff
@@ -165,6 +227,15 @@ def ensure_view_change_complete(looper, nodes, exclude_from_check=None,
     ensure_all_nodes_have_same_data(looper, nodes, customTimeout)
 
 
+def ensure_view_change_complete_by_primary_restart(
```

Comment:
Do we use this method?

Reply:
no, it was debugged but is not actually used for now, added just for possible future usage